packages/kbn-apm-synthtrace-client/src/lib/apm/apm_fields.ts

```diff
@@ -115,6 +115,7 @@ export type ApmFields = Fields<{
     'http.response.status_code': number;
     'kubernetes.pod.name': string;
     'kubernetes.pod.uid': string;
+    'labels.name': string;
```

Comment:
Are you planning to add synthtrace scenarios to this PR?

Reply:
I could add a new scenario that matches the events i defined for the API test, Might be useful for development and testing.

Reply:
I added the `service_map` scenario to synthtrace that has some resemblance to the hipster store:
![Screenshot 2023-02-02 at 11 27 10 PM](https://user-images.githubusercontent.com/1967266/216512868-02406e65-7d18-4313-aea0-395f50c6a574.png)